client/src/components/game/gui/panels/map/MapPanel.scss

```diff
@@ -4,5 +4,18 @@
     background-color: $colourDB32White;
 
     .inner-cont {
+        height: 100%;
+        width: 100%;
     }
-}
\ No newline at end of file
+
+    .leaflet-container {
+        border-radius: 1.5rem;
```

Comment:
This border rounding trick to hide the overlapping can be fixed like so:

```css
.map-panel {
    background-color: $colourDB32White;
    pointer-events: none; <-- allow pointer events through
    
    .leaflet-container {
        pointer-events: auto; <-- catch pointer events for this element
        z-index: -1; <-- move the element below the rest of the stuff around the edge of the panel template.
        height: 100%;
        width: 100%;
    }

    .leaflet-div-icon {
        background: transparent;
        border: none;
    }
}
```

Reply:
Oh thanks 💯, I couldn't figure this out on my own.